doc/LANGUAGE.md

```diff
@@ -345,26 +345,36 @@ Instruction                 | Flags   | Description
 
 The `const` instruction is used to introduce a constant value into the IR. The first version constructs a constant integer value, the second a constant integer signal, and the third a constant time value.
 
-    %result = const iN <int>
-    %result = const iN$ <int>
     %result = const time <time>
+    %result = const iN <int>
+    %result = const nN <enum>
+    %result = const lN <logic>
 
+- `time` is a time literal such as `1s`, `1s 2d`, or `1s 2d 3e`, where the real component may carry an SI suffix such as `as`, `fs`, `ps`, `ns`, `us`, `ms`, `s`.
 - `int` is an integer literal such as `0b0101`, `0o1247`, `129`, or `0x14F3E`
-- `time` is a time literal such as `1s`, `1s 2d`, or `1s 2d 3e`, where the real component may carry an SI suffix such as `as`, `fs`, `ps`, `ns`, `us`, `ms`.
+- `enum` is a decimal integer literal which ranges from `0` to `N-1`
```

Comment:
We could add here that `enum` allows for the same binary, octal, decimal, and hexadecimal literals as `int`, but range-limited.

Reply:
Sounds good to me. Would simply removing the word "decimal" from the sentence achieve this or should we be more explicit than that?

Reply:
Maybe refer to the syntax for the `int` case?